src/sentry/api/serializers/models/plugin.py

```diff
@@ -61,6 +61,14 @@ def serialize(self, obj, attrs, user):
                 'url': six.text_type(obj.author_url)
             }
 
+        if obj.description:
+            d['description'] = six.text_type(obj.description)
+
+        if obj.resource_links and len(obj.resource_links):
```

Comment:
i don't think you need this `len` because empty lists/tuples/dicts are falsey in python